runtime/src/bank/tests.rs

```diff
@@ -983,10 +973,7 @@ fn test_rent_eager_collect_rent_in_partition() {
         bank.get_account(&rent_exempt_pubkey).unwrap().rent_epoch(),
         RENT_EXEMPT_RENT_EPOCH
     );
-    assert_eq!(
-        bank.slots_by_pubkey(&rent_due_pubkey),
-        vec![genesis_slot, some_slot]
-    );
+    assert_eq!(bank.slots_by_pubkey(&rent_due_pubkey), vec![genesis_slot]);
```

Comment:
We no longer rewrite the rent_due account when skip rewrites is enabled for rent_due account. 
rent_due account won't be rent collected after `disable rent collection` feature and rent_due account's rent_epoch won't be updated because it is not rent exempted. So the account won't change.